src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -279,11 +279,27 @@ private String getErrorMessage(String service, String existingCaseId, String exc
             return Optional.empty();
         } catch (FeignException.UnprocessableEntity exception) {
             String msg = format(
-                "Service returned 422 Unprocessable Entity response "
+                "CCD returned 422 Unprocessable Entity response "
                     + "when trying to update case for %s jurisdiction "
                     + "with case Id %s "
                     + "based on exception record with Id %s. "
-                    + "Service response: %s",
+                    + "CCD response: %s",
+                exceptionRecord.poBoxJurisdiction,
+                existingCase.getId(),
+                exceptionRecord.id,
+                exception.contentUTF8()
+            );
+            log.error(msg, exception);
+
+            return Optional.of(msg);
```

Comment:
We can't return the error to the caseworker in this case. Instead, the processing should fail, resulting in the orchestrator returning a 500 response to CCD.